### PR TITLE
discussion section

### PR DESCRIPTION
**Pull request by Farnazmdi**
Wrote an outline for discussion (just some titles above each paragraph)
There are some sentences in the second paragraph that probably should be in the first paragraph, but I was not sure if we need to make the first paragraph that long... Mostly they are references from relevant literature.

Do you mind skimming this new discussion section and letting me know if it is close, or worse, or something obvious missing? @aarmey 
I tried to include more non-model content, but I am not sure what I have missed.

### CODE REVIEW

**Comment by aarmey** [comment]
@Farnazmdi, this is not an [outline](https://en.wikipedia.org/wiki/Outline_(list)).

**Comment by aarmey** [comment]
Specifically, I'd like us to organize this purely in list form—no paragraphs—before turning it into paragraphs. This process will force it to become more organized.

**Comment by Farnazmdi** [comment]
I updated it and now it includes a list for each section.
I understand it is still focused on the model. I need to find some literature to relate the findings from experimental data to the literature.

File: manuscript/04.discussion.md [hunk 1]
```diff
@@ -1,11 +1,51 @@
...
+
+Additional findings and how this fits to existing literature:
+
+1. tHMM shows high sensitivity for under-represented sub-populations.
+
+2. tHMM has proved to be more accurate than K-means clustering which shows the importance of accounting for history of cells and their relations.
+
+3. tHMM properly handles time-censorship using survivoral function of distributions, which is barely the case in the literature.
+
+4. Insightful findings from lapatinib treatment fitting, and comparing to some literature about effects of lapatinib on breast cancer cells:
```
Comment (aarmey) [inline manuscript/04.discussion.md:21]: I'd separate #4/5 into their own paragraph about what we learn about these drugs.

File: manuscript/04.discussion.md [hunk 2]
```diff
@@ -1,11 +1,51 @@
...
+
+1. Discritizing the state, while in reality they are coninuous variables, is a challenging assumption which may cause inaccuracy to some extent.
+
+2. tHMM cannot perform with high accuracy when the states are too close.
+
+3. In the current setting, the measurements are assumed to be independent. If observations are not independent variables, we should account for that in the likelihood function.
+
+4. The experiment time is limited, and we may not have captured slow-fluctuations. Also, it is hard to collect this type of data for longer times due to experimental limitations.
+
+5. Without further experiments, the biological interpretations of each state is not clear.
```
Comment (aarmey) [inline manuscript/04.discussion.md:41]: I'd say *molecular* interpretations of state are unclear. Phenotype is biology!

**Comment by aarmey** [comment]
This is a *great* outline. This, turned into paragraph form, will make a great discussion section.

**Comment by Farnazmdi** [comment]
@aarmey do you mind taking a look at this again? I tried to find literature about the effects of gemcitabine and lapatinib on au565 cells, or even searched HER2-overexpressing cells, but didn't find much. Are those two paragraphs enough about the comparison and the effects?

**Comment by aarmey** [comment]
I'll work on edits off this.

**Pull request merged**